device-types/Cisco/N9K-C9332D-GX2B.yaml

```diff
@@ -0,0 +1,87 @@
+---
+manufacturer: Cisco
+model: Nexus 9332D-GX2B
+part_number: N9K-C9332D-GX2B
+slug: n9k-c9332d-gx2b
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: console
+    type: rj-45
+power-ports:
+  - name: Power Supply 1
+    type: iec-60320-c16
+  - name: Power Supply 2
+    type: iec-60320-c16
```

Comment:
In my opinion, these should be in a module and these should be replaced with module bays